packages/angular/cli/upgrade/version.ts

```diff
@@ -70,9 +94,10 @@ export class Version {
       return;
     }
 
-    if (!angularVersion.isGreaterThanOrEqualTo(new SemVer('5.0.0'))) {
+    if (!angularVersion.satisfies(supportedAngularSemver)) {
```

Comment:
looks like the pre-release option is not used here?

Reply:
`angularVersion` is an instance of `Version`. I added it directly in the `satisfies` method:
```
return this._semver !== null && satisfies(this._semver, other, { includePrerelease: true });
```